proto/system_config.proto

```diff
@@ -138,6 +138,9 @@ message DaqConfig {
 
   // Path to include file
   string include = 56;
+
+  // The maximum time in seconds between messages from faucet
+  int32 faucet_event_timeout_sec = 57;
```

Comment:
I'd like to keep the concept of "faucet" out of the exposed config, so maybe just call this "event_timeout_sec"

Reply:
I thought about it but if I were to leave out faucet, this configuration is overly vague. It’s like renaming faucet_event_client to just event_client

Reply:
The configuration is an external interface specification, and really shouldn't be bleeding the fact that "faucet" is used internally.  This is different than the code (faucet_event_client) since that's internal to the system (implementation not interface). I'm more concerned about bleeding the abstraction than I am it being overly vague. The concept of an "event timeout" has meaning even when it's not faucet (e.g. could map to whatever system it's assumed to be getting such events from).